content/documentation/3-developer-guides/3-batch/1-simple-task.md

```diff
@@ -11,31 +11,293 @@ In this guide we will develop a batch application using Spring Cloud Task and de
 We will start from initializr and create two Spring Cloud Batch applications.
 Note for CF we need a manifest, for k8s we need a service/deployment yaml.
 
+NOTE: All code for this project can be found here: https://github.com/cppwfs/edutasksamples
+
 ## Development
 
-Explaine what we are doing to do development wise.
+Imagine you are working for a cell phone data provider and you are responsible for writing the solution that will create the billing statements for the customers. The usage data that you will be using will be stored in JSON files stored in a file on your file system. Your solution will be responsible for pulling the data from these files, storing the raw data into a table in your database and then generate a price from this usage data and storing it into a billing_statement table.  
+We could implement this entire solution into a single Spring Boot Application that utilizes Spring Batch, however for this example we will break up the solution into 2 phases:
+
+1. BillUsage will be a Spring Boot application using Spring Cloud Task that will read data from JSON files from the classpath and put the data into the `BILL_USAGE` table.
+1. BillRun will be a Spring Boot application using Spring Cloud Task and Spring Batch that will read data from `BILL_USAGE` table and price the each row and put the resulting data into the `BILL_STATEMENTS` table.
+
+'For this section we will create a Spring Cloud Task/Boot application that will read JSON files containing customer usage data from a S3 bucket and write the raw data to a table in the database.  
+The data contained in the files will be: id, customer first name , customer last name, cellular minutes, and data usage. The format of the data looks like this
+
+```
+[{"id":"16","firstName":"jane","lastName":"doe","minutes":"500","dataUsage":"1000"},
+{"id":"20","firstName":"mary","lastName":"jones","minutes":"650","dataUsage":"1500"}]
+```
 
-Create a simple task (timestamp)
+This usage data will then be inserted into a `BILL_USAGE` table that looks like the following:
+
+```
+CREATE TABLE BILL_USAGE
+(
+   id int,
+   first_name varchar(50),
+   last_name varchar(50),
+   minutes int,
+   data_usage int
+)
+```
```

Comment:
I'm a fan of communicating with developers in code. Images should be for things that are harder to express in code (an entity relationship diagram that shows the connections between a set of tables, for example). Also, I've done enough SQL work that an SQL table definition pretty much is an image for me. I doubt that I'm unique in that regard. Just my $.02 rather than a firm recommendation for this one, but I'd do this bit as SQL code.

Reply:
I've applied the image for now.  Let's see how it looks.